Source/source/GPS.cpp

```diff
@@ -58,11 +60,6 @@ string toStringTime(struct minmea_time *time) {
 	return timeString.str();
 }
 
-string get_message() {
-	string message; // set message
-	return message;
```

Comment:
This will have to be reimplemented once message receiving code is finished from the OS team. Although since it is just methods from another class, this method will theoretically become the “main” method of our program

Reply:
Ahh, I misunderstood its purpose then, when I removed it. At the time, I thought it was a placeholder for getting a "message" from the GPS, which made it redundant, but what you say makes a lot more sense!